docs/api/specs/desktopAgentCommunicationProtocol.md

```diff
@@ -0,0 +1,457 @@
+---
+id: desktopAgentCommunicationProtocol
+sidebar_label: Desktop Agent Communication Protocol 
+title: Desktop Agent Communication Protocol  (next)
+---
+
+# Desktop Agent Communication Protocol (DACP)
```

Comment:
Is this "# Desktop Agent Communication Protocol (DACP)" line actually (wrongly) overriding a title that would otherwise be set via the title property in the header?
It's just that I noticed that other pages _appear_ to be generating a title from the title property, rather than using "# XXX". Also:
- if the "# XXX" line is in fact overriding, that's probably why the preview does not show the standard "(next)" suffix, which I believe you mentioned is important as it will ultimately get auto-replaced by "(2.2)" on merge/release
- the current "(DACP)" suffix in the title shown in the previous is inconsistent with the webConnectionProtocol.md page, which I noticed does **not** include a "(WCP)" suffix.  Which will also be sorted by fixing the above.